backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -205,11 +206,13 @@ public static List<FeedbackMessage> validateEmail(ValueOrError input) {
   }
 
   public static Map<String, String> getNextRow(MappingIterator<Map<String, String>> valueIterator)
-      throws IllegalArgumentException {
+      throws CsvProcessingException {
```

Comment:
I think we'll need to make adjustments to `PatientBulkUploadService` and `PatientBulkUploadFileValidator` with this change:
https://github.com/CDCgov/prime-simplereport/blob/f68375bd8fd7bfd55b42e1dc3f9c8a899d29f332/backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java#L146-L146

Up to you if you'd like to include those changes in this PR or just open a ticket for them, though - I don't want to hold this PR up.

Reply:
Yeah this is why I was asking for any patient bulk upload files you had for testing.

Reply:
update made to the file validator - I think that's the only update the patient side needs based on my testing